maui-toolkit/Bottom-Sheet/BottomSheet-Customization.md

```diff
@@ -195,65 +166,66 @@ Content = bottomSheet;
 {% endhighlight %}
 {% endtabs %}
 
-## Content Width Mode
-The `ContentWidthMode` property allows you to adjust the width of the `BottomSheet`. The default value is `Full`. This property accepts the following values:
+## Adjust the Height
+### FullExpandedRatio
 
-* **Full** - The sheet will occupy the entire screen width.
-* **Custom** - The sheet will adjust its width based on the value set in the `BottomSheetContentWidth` property.
+The `FullExpandedRatio` property adjusts the height of the `BottomSheet` when it is in the `FullExpanded` state. The default value is `1`. You can set a value between 0.1 and 1 to adjust the height. This property works only when the sheet's `State` is `FullExpanded`.
 
 {% tabs %}	
 {% highlight xaml %}
 
-<bottomSheet:SfBottomSheet x:Name="bottomSheet" ContentWidthMode="Full">
-    <bottomSheet:SfBottomSheet.BottomSheetContent>
-        <Grid/>
-    </bottomSheet:SfBottomSheet.BottomSheetContent>
+<bottomSheet:SfBottomSheet x:Name="bottomSheet" FullExpandedRatio="0.8" State="FullExpanded">
+ <bottomSheet:SfBottomSheet.BottomSheetContent>
+     <Grid/>
+ </bottomSheet:SfBottomSheet.BottomSheetContent>
 </bottomSheet:SfBottomSheet>
 	
 {% endhighlight %}
 {% highlight c# %}
 
 SfBottomSheet bottomSheet = new SfBottomSheet();
-bottomSheet.ContentWidthMode = BottomSheetContentWidthMode.Full;
+bottomSheet.FullExpandedRatio = 0.8;
+bottomSheet.State = BottomSheetState.FullExpanded;
 Grid grid = new Grid();
 bottomSheet.BottomSheetContent = grid;
 Content = bottomSheet;
 
 {% endhighlight %}
 {% endtabs %}
 
-## Bottom Sheet Content Width
-The `BottomSheetContentWidth` property allows you to adjust the width of the `BottomSheet`, when it is in `Custom` content width mode. The default value is `300`.
+### HalfExpandedRatio
+
+The `HalfExpandedRatio` property adjusts the height of the `BottomSheet` when it is in the `HalfExpanded` state. The default value is `0.5`. You can set a value between 0.1 and 0.9 to adjust the height. This property works only when the sheet's `State` is `HalfExpanded`.
```

Comment:
The `HalfExpandedRatio` property adjusts the height of the `BottomSheet` when it is in the `HalfExpanded` state. The default value is `0.5`. You can set a value between 0.1 and 0.9 to adjust the height.

Reply:
Updated the content.